ui-admin/src/study/StudySettings.tsx

```diff
@@ -63,10 +62,18 @@ export function StudyEnvConfigView({ studyEnvContext, portalContext }:
           onChange={e => updateConfig('password', e.target.value)}/>
       </label>
     </div>
+    <div>
+      <label className="form-label">
+        accepting enrollment <InfoPopup content={`Uncheck this to hide the study from participants who have not
```

Comment:
Nit: I am unclear on why the choices on this form (mostly) do not capitalize the first letter. If that's the norm, fine, but otherwise it projects an unfinished feel.
(Also, not due to this PR, but it seems like there should be some padding between "accepting registration" and the following checkbox. In general, it feels like there could be a bit more vertical spacing between choices to help readability and to look more polished. Likewise, there is inconsistency in wording of the two sections: "Ourhealth study configuration" and "Website configuration (Ourhealth).)